astropy/io/fits/tests/test_core.py

```diff
@@ -782,10 +763,9 @@ def test_open_bzipped_from_handle(self):
     @pytest.mark.skipif(not HAS_BZ2, reason='Python built without bz2 module')
     def test_detect_bzipped(self):
         """Test detection of a bzip2 file when the extension is not .bz2."""
-        with ignore_warnings():
```

Comment:
Note: I don't know were warnings are ignored blindly and turns out they are not necessary anyway.